crates/biome_service/src/diagnostics.rs

```diff
@@ -341,6 +343,30 @@ pub enum SearchError {
 )]
 pub struct InvalidPattern;
 
+#[derive(Debug, Serialize, Deserialize)]
+pub struct QueryError(pub String);
+
+impl Diagnostic for QueryError {
+    fn category(&self) -> Option<&'static Category> {
+        Some(category!("search"))
+    }
+
+    fn severity(&self) -> Severity {
+        Severity::Error
+    }
+
+    fn message(&self, fmt: &mut biome_console::fmt::Formatter<'_>) -> std::io::Result<()> {
+        fmt.write_markup(markup! { "Error executing the Grit query: "{{&self.0}} })
+    }
+
+    fn verbose_advices(&self, visitor: &mut dyn Visit) -> std::io::Result<()> {
+        visitor.record_log(
+            LogCategory::Info,
+            &markup! { "For reference, please consult: https://docs.grit.io/language/syntax" },
```

Comment:
You can use `<Hyperlink href="">please consult the official grit syntax page.</Hyperlink>` to create a clickable link.

Reply:
Addressed in https://github.com/biomejs/biome/pull/3288/commits/240455ba282b4061510af5821e1c0bc3ee27ed32.